index.html

```diff
@@ -88,49 +96,224 @@
       update();
     });
   }
-
   function update() {
     // TRANSFORM
     txData = data.slice();
-
     if (options.yearRange !== 'all') {
       var yearRange = options.yearRange.split('-');
       txData = txData.filter(function (d) {
         return d.year >= yearRange[0] && d.year <= yearRange[1];
       });
     }
-
     // SCATTERPLOT
+    var points = svg.selectAll('.point')
+      .data(data);
+    points.enter().append('circle')
+      .attr('class', 'point')
+      .attr('r', 5);
+    points
+      .attr('cx', function (d) { return x(d.unemployment); })
+      .attr('cy', function (d) { return y(d.employment); });
+    points.exit().remove();
+    
+    
+
+    // Creating (printing the Axxis)
+    //Create X axis
+      svg.append("g")
+        .attr("class", "axis")
+        .attr("transform", "translate(0," + (height - xpadding) + ")")
+        .call(xAxis)
+        .append("text")
+         .text("employment (%)")
+         .attr("x", width - 63)
+         .attr("y", 0)
+         .attr("font-family", "sans-serif")
+         .attr("font-size", "11px")
+         .attr("fill", "red");
+      
+      //Create Y axis
+      svg.append("g")
+        .attr("class", "axis")
+        .attr("transform", "translate(" + ypadding + ",0)")
+        .call(yAxis)
+        .append("text")
+          .text("unemployment (%)")
+          .attr("x", 0)
+          .attr("y", 0)
+          .attr("font-family", "sans-serif")
+          .attr("font-size", "11px")
+          .attr("fill", "blue");
+           
+      
 
+    //CALLING THE GRID
+    svg.append("g")         
+        .attr("class", "grid")
+        .attr("transform", "translate(0," + height + ")")
+        .call(make_x_axis()
+            .tickSize(-height, 0, 0)
+            .tickFormat("")
+        )
+
+    svg.append("g")         
+        .attr("class", "grid")
+        .call(make_y_axis()
+            .tickSize(-width, 0, 0)
+            .tickFormat("")
+        )
+    
     // HISTOGRAMS
 
-    // REGRESSION LINE
+    var formatCount = d3.format(",.0f");
+
+    //var margin = {top: 10, right: 30, bottom: 30, left: 30},
+        //width = 960 - margin.left - margin.right,
+        //height = 500 - margin.top - margin.bottom;
+
+    //Alex Answer: Yes, you do need to recreate the scale for the height of the rects in the histogram. Now, for each histogram, you are going to want a linear scale. This scale will have range and a domain, like other scales. 
+    //The domain should refer to your input data: What are the lowest and highest values that are important to the height of a histogram?
+    //The range should still be for the range of the output space, starting at 0, and going to the maximum height you would like the bars to going.
+    //Note that in the image Chris attached to the assignment, there is only one axis for each variable. So you don't have to recreate the axes.
+    
+    
+    xh = d3.scale.linear()
+        .rangeRound([0, width]);
```

Comment:
Note that `.rangeRound()` rounds the output value to the nearest even pixel. Useful for getting sharp edges, but perhaps less accurate to the data. I'd be okay with using that for something like the width of the histogram bars, so that their left and right edges are snapped to pixels. Probably less of a good idea for the height of the bars.